.github/dependabot.yml

```diff
@@ -0,0 +1,16 @@
+version: 2
+
+updates:
+  - package-ecosystem: "github-actions"
```

Comment:
Don't we need `npm` here as well? Seems like this would skip updates to npm packages.

Reply:
I didn't know if you wanted to update packages so I started with actions. 😄

Reply:
I can add npm packages to this if you would like or make a separate PR for it.